code/modules/admin/topic.dm

```diff
@@ -1,3 +1,6 @@
+GLOBAL_LIST_EMPTY(admins_topic_datum)

```

Comment:
should probably be protected

Reply:
That I shall do when I come back to this.